conformance/tests/aliases_newtype.py

```diff
@@ -14,6 +14,8 @@
 
 assert_type(UserId(5) + 1, int)
 
+UserId.mro()  # E: NewType is not an instance of `type`
```

Comment:
By my reading of the spec, this shouldn't result in a type error. Can you point to the text in the spec that would indicate that it should? All conformance tests should be motivated by the language in the typing spec. If you think that there's a hole in the typing spec, please work through the process to change the spec.

Reply:
great, this has been updated and pushed